frontend/packages/dev-console/src/components/helm/helm-utils.ts

```diff
@@ -106,7 +106,7 @@ export const getHelmActionConfig = (
         subTitle: 'The helm chart will be installed using the YAML shown in the editor below.',
         helmReleaseApi: `/api/helm/chart?url=${chartURL}`,
         fetch: coFetchJSON.post,
-        redirectURL: `/topology/ns/${namespace}`,
+        redirectURL: `/topology/ns/${namespace}/graph`,
```

Comment:
Another reason for using the modal!
If the user prefers using the list view, we're dumping them into the graph view and ignoring their preference.

cc @serenamarie125 @Veethika

Reply:
I agree.